master/internal/sproto/kubernetes_resource_manager.go

```diff
@@ -20,6 +20,22 @@ type (
 		PodID container.ID
 	}
 
+	// PreemptTaskPod notifies the pods actor to preempt a pod.
+	PreemptTaskPod struct {
+		PodName string
+	}
+
+	// ChangePriority notifies the pods actor to preempt a pod
+	ChangePriority struct {
+		PodID container.ID
```

Comment:
question: does it need to specify a priority?

Reply:
nope! We just need to notify the pods actor to kill the pod. After that, we automatically re-submit under a new priority.